ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcTlsConfig.java

```diff
@@ -75,14 +77,23 @@ public boolean getMtlsEnabled() {
   }
 
   public GrpcTlsConfig(PrivateKey privateKey, X509Certificate certChain,
-      X509Certificate trustStore, boolean mTlsEnabled) {
+      List<X509Certificate> trustStore, boolean mTlsEnabled) {
     this.privateKey = privateKey;
     this.certChain = certChain;
     this.trustStore = trustStore;
     this.mTlsEnabled = mTlsEnabled;
     this.fileBasedConfig = false;
   }
 
+  public GrpcTlsConfig(PrivateKey privateKey, X509Certificate certChain,
+      X509Certificate trustStore, boolean mTlsEnabled) {
+    this.privateKey = privateKey;
+    this.certChain = certChain;
+    this.trustStore = Arrays.asList(trustStore);
```

Comment:
Suggestion: use `Collections.singletonList` instead of `Arrays.asList`.

Reply:
Done